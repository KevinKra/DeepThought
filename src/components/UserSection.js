import React from "react";
import "../styles/_UserSection.scss";
import userCard from "./UserCard";
import UserCard from "./UserCard";
class UserSection extends React.Component {
  render() {
    return (
      <section className="userSection">
        <h2>UserName</h2>
        <article>
          <UserCard />
          <UserCard />
          <UserCard />
        </article>
      </section>
    );
  }
}

export default UserSection;
