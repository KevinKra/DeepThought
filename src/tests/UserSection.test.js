import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UserSection from "../components/UserSection";

describe("<UserSection />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<UserSection />);
  });
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
