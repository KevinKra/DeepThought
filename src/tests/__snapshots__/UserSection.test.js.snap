// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserSection /> should match the snapshot 1`] = `
<section
  className="userSection"
>
  <h2
    className="userGreeting"
  >
    Welcome, undefined!
  </h2>
</section>
`;
